src/App.vue

```diff
@@ -3,6 +3,7 @@
 </template>
 
 <script>
+// TODO why we need this component?
```

Comment:
We creating the root component in `main.js`. The root component used the `App.vue` component, and this component do nothing except the same thing- use the next component.

So I'm removing this.

Reply:
Actually we doing so many changes, no need to describe each one.